vsts/pipelines/PlatformBinaries/dotnetcore.yml

```diff
@@ -1,6 +1,12 @@
 variables:
   - group: Oryx
 
+parameters:
+- name: destinationStorageAccount
```

Comment:
nit: it's probably a lot of copy-paste, but I would defer to having this parameter called `destinationStorageAccountName`, just so we have the differentiation between the storage account resource name and the URL of the storage account.